### PR TITLE
[Community] Add member profile

### LINKED ISSUE #2879: [Community] Member Profile: Oyin Dawodu and Ednah Akoth
**Current Behavior**

@mzoyinda and @Ed-Neema has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @mzoyinda and @Ed-Neema as a contributors and community members by creating a profile on https://layer5.io/community/members.

- Github: https://github.com/mzoyinda
- Twitter: https://twitter.com/oyindawodu
- Linkedin: https://www.linkedin.com/in/oyin-dawodu-989434b6/
- Profile picture: https://drive.google.com/file/d/1c7aoCjEjwP1XYBFaohz0U9_IiJ1qiC2j/view?usp=sharing

Github: @Ed-Neema
Twitter: edmucious
Linkedin: ednah-akoth-7515a0207
Profile picture: https://drive.google.com/file/d/1gUlOvQFQxm7H2lgIklDduWv_f8ykl5_l/view?usp=sharing 

A detailed explanation of how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by VaibhavArora19**
Signed-off-by: Vaibhav Arora <aroravaibhav817@gmail.com>

**Description**

This PR fixes #2879

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c2f65f3a3f6016448b42cfc4214c82e61b3654d0 at: https://62a38c850428e305dab424b2--layer5.netlify.app

File: src/collections/members/ednah-akoth/index.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Ednah Akoth
+position: Contributor, Community Member
```
Comment (Chadha93) [inline src/collections/members/ednah-akoth/index.mdx:3]:
```suggestion
position: Contributor
```

File: src/collections/members/oyin-dawodu/index.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: Oyin Dawodu
+position: Contributor, Community Member
```
Comment (Chadha93) [inline src/collections/members/oyin-dawodu/index.mdx:3]:
```suggestion
position: Contributor
```

**Review by YashKamboj — APPROVED** [review]
LGTM!!!

File: src/collections/members/ednah-akoth/index.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+image_path: ./ednah-akoth.jpg
+github: Ed-Neema
+twitter: edmucious
+linkedin: ednah-akoth-7515a0207
+location: Kigali, Rwanda
```
Comment (Chadha93) [inline src/collections/members/ednah-akoth/index.mdx:8]: @Ed-Neema would you please provide a bio, so @VaibhavArora19 could add it.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@VaibhavArora19 good work, some minors to look. :)

**Comment by YashKamboj** [comment]
missed it 😅

**Comment by l5io** [comment]
🚀 Preview for commit b508350028535eb1ef9e45af6eeed55ba2cb5fe3 at: https://62a392f811291e007c755be5--layer5.netlify.app

File: src/collections/members/ednah-akoth/index.mdx (hunk 3 above)
Comment (VaibhavArora19) [reply src/collections/members/ednah-akoth/index.mdx:8]: I think she is not active on github lately

Comment (Ed-Neema) [reply src/collections/members/ednah-akoth/index.mdx:8]: Hello @Chadha93 Apologies for the delay in response. Here's my bio write-up

I am an an undergrad Computer Science with inclinations towards Frontend development (Vue, tailwindcss, Laravel php) and Data Science. I enjoy learning new technologies and applying them to solve problems and build robust products. I have grown to relish open source contribution, not only as a means of networking with brilliant professionals and building/improving products with a global reach, but also as a means of spurring my own technical growth. Besides coding, you’d typically find me jamming on my guitar or eating.

Comment (VaibhavArora19) [reply src/collections/members/ednah-akoth/index.mdx:8]: Thanks @Ed-Neema

**Comment by l5io** [comment]
🚀 Preview for commit 81bd5f68e67c96b15814d69244e5f597643df06e at: https://62a577dfaf906f2238619cca--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**